bin/mail/2016-08-09-hoc-latam/send-real-emails

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+#3661
```

Comment:
Just saw your change. Ignore latter part of comment. Is limit just a precaution? How come it's not more exact like limit 9055 or 9056? Does it count retries?